packages/next/client/image.tsx

```diff
@@ -369,6 +369,15 @@ export default function Image({
     }
   }
 
+  if (layout !== 'fill' && !unsized) {
```

Comment:
```suggestion
  if (layout === 'intrinsic' || layout === 'fixed') {
```

I think we can only set this attribute when `heightInt` is defined.

Furthermore, I think it only works with intrinsic and fixed because the `height` prop for `layout=responsive` is not actually the height, its just used to calculate the aspect ratio. So the height of `layout=responsive` is really dictated by the parent element (similar to `layout=fill`).

Therefore its best to only use this new CSS property for intrinsic/fixed where we know the height.

Reply:
Good catch on `layout="responsive"`! Change committed.